hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMaster.java

```diff
@@ -4095,4 +4095,17 @@ public String getClusterId() {
   public Collection<ServerName> getLiveRegionServers() {
     return regionServerTracker.getRegionServers();
   }
+
+  @RestrictedApi(explanation = "Should only be called in tests", link = "",
+      allowedOnPath = ".*/src/test/.*")
+  void setLoadBalancer(RSGroupBasedLoadBalancer loadBalancer) {
```

Comment:
Good to add explicit comments for both of these methods that they are only supposed to be used by test code.